app/src/services/conditional_token.ts

```diff
@@ -73,12 +75,14 @@ class ConditionalTokenService {
     return this.contract.balanceOf(ownerAddress, positionId)
   }
 
-  getQuestionId = async (conditionId: string, provider: any): Promise<string> => {
+  getQuestionId = async (conditionId: string): Promise<string> => {
     const filter: any = this.contract.filters.ConditionPreparation(conditionId)
 
-    filter.fromBlock = '0x1'
-
-    const logs = await provider.getLogs(filter)
+    const logs = await this.provider.getLogs({
+      fromBlock: 1,
+      toBlock: 'latest',
+      ...filter,
```

Comment:
Put `fromBlock` and `toBlock` after `...filter`.

`filter` probably doesn't have those fields, but it's better to err on the safe side.